secant/Dependencies/ZcashSDKEnvironment/ZcashSDKEnvironmentLiveKey.swift

```diff
@@ -14,24 +14,23 @@ extension ZcashSDKEnvironment: DependencyKey {
     static let liveValue = Self(
         defaultBirthday: BlockHeight(ZcashSDKConstants.defaultBlockHeight),
         endpoint: LightWalletEndpoint(
-            address: ZcashSDKConstants.endpointMainnetAddress,
+            address: ZcashSDKConstants.endpointTestnetAddress,
             port: ZcashSDKConstants.endpointPort,
             secure: true,
             streamingCallTimeoutInMillis: ZcashSDKConstants.streamingCallTimeoutInMillis
         ),
-        isMainnet: { true },
         lightWalletService: LightWalletGRPCService(
             endpoint: LightWalletEndpoint(
-                address: ZcashSDKConstants.endpointMainnetAddress,
+                address: ZcashSDKConstants.endpointTestnetAddress,
                 port: ZcashSDKConstants.endpointPort,
                 secure: true,
                 streamingCallTimeoutInMillis: ZcashSDKConstants.streamingCallTimeoutInMillis
             )
         ),
         memoCharLimit: 512,
```

Comment:
There should be an SDK constant representing this value

Reply:
I don't understand. `ZcashSDKConstants` is from SDK.

Reply:
There isn't. Or at least I didn't find it. Even SDK sample app defines this on it's own.

Reply:
Oh I recall why that is.

The correct way of handling memo limit is through byte size. that is `MemoBytes.capacity` 

````Swift
public struct MemoBytes: Equatable {
    public enum Errors: Error {
        /// Invalid UTF-8 Bytes where detected when attempting to create a Text Memo
        case invalidUTF8
        /// Trailing null-bytes were found when attempting to create a Text memo
        case endsWithNullBytes
        /// the resulting bytes provided are too long to be stored as a Memo in any of its forms.
        case tooLong(Int)
    }

    public static let capacity: Int = 512
````

Having a char limit is not safe unless UTF-8 is enforced as input encoding. Also There can be compound characters like Emojis that alter the char count.